documentation/source/users/cantherm/input.rst

```diff
@@ -13,12 +13,13 @@ Each section is made up of one or more function calls, where parameters are
 specified as text strings, numbers, or objects. Text strings must be wrapped in
 either single or double quotes.
 
-The following is a list of all the functions of a CanTherm input file for thermodynamics and high-pressure limit kinetics computations:
+The following is a list of all the elements of a CanTherm input file for thermodynamics and high-pressure limit kinetics computations:
```

Comment:
I'm uncertain regarding this terminology change: "elements" sound like chemical elements. By the way, the table below still says `Function`, as well as other references in the documentation below. Since it's a python file, it's OK to call them functions. But we could consider "components" as well.

Reply:
The reason I changed it is because many of the options aren't functions, they're assignment statements. That was probably changed in the past, but the documentation never got updated. I thought I went through and checked for all occurrences of "function", but evidently I wasn't thorough enough. I agree that "elements" could be misleading. I'll give it a different name.